main.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+function daysInYear(int) {
+  const year = int;
+
+  if (year % 1 === 0 && typeof year === 'number') {
+    function daysInYear(year) 
+    {
+      
+      return isLeapYear(year) ? 366 : 365;
+    }
+
+    function isLeapYear(year) {
+        return year % 400 === 0 || (year % 100 !== 0 && year % 4 === 0);
+    }
+    return daysInYear(year);
+  }
+  throw 'exception';
```

Comment:
will be better to always throw instances of the Erro to be consistent with default errors